src/test/java/org/gridsuite/modification/server/modifications/OperatingStatusModificationLockoutLineTest.java

```
@@ -79,35 +80,53 @@ protected void assertAfterNetworkModificationDeletion() {
+        assertNotNull(line);
+        modificationInfos.setEquipmentId("nonFictionalDisconnect");
+        modificationInfos.setAction(OperatingStatusModificationInfos.ActionType.LOCKOUT);
+        String modificationJson = mapper.writeValueAsString(modificationInfos);
+        assertNull(getNetwork().getLine("nonFictionalDisconnect").getExtension(OperatingStatus.class));
+
+        mockMvc.perform(post(getNetworkModificationUri()).content(modificationJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+        assertNotNull(getNetwork().getLine("nonFictionalDisconnect").getExtension(OperatingStatus.class));
+
+    }
+
     @Test
     public void testCreateWithErrors() throws Exception {
         // line not existing
         OperatingStatusModificationInfos modificationInfos = (OperatingStatusModificationInfos) buildModification();
         modificationInfos.setEquipmentId("notFound");
         String modificationJson = mapper.writeValueAsString(modificationInfos);
-        mockMvc.perform(post(getNetworkModificationUri()).content(modificationJson).contentType(MediaType.APPLICATION_JSON))
-            .andExpect(status().isOk());
-        assertNull(getNetwork().getLine("notFound"));
-        assertLogMessage(new NetworkModificationException(EQUIPMENT_NOT_FOUND, "notFound").getMessage(),
-                modificationInfos.getErrorType().name(), reportService);
-
-        // modification action empty
-        modificationInfos.setEquipmentId("line2");
-        modificationInfos.setAction(null);
-        modificationJson = mapper.writeValueAsString(modificationInfos);
-        mockMvc.perform(post(getNetworkModificationUri()).content(modificationJson).contentType(MediaType.APPLICATION_JSON))
-            .andExpectAll(
-                    status().isBadRequest(),
-                    content().string(new NetworkModificationException(OPERATING_ACTION_TYPE_EMPTY).getMessage())
-            );
```

Put back this test?

---

Add:
* one test with a fictitious switch => it should throw an exception
* one test with a non breaker switch

---

done